multimedia/VLC/Portfile

```diff
@@ -40,434 +30,673 @@ subport             lib${name} {
+
+                    # These are already in ${prefix}, so we don't need to bundle them as well
+                    delete ${destroot}${applications_dir}/VLC.app/Contents/Frameworks
+                    delete ${destroot}${applications_dir}/VLC.app/Contents/lib
+
+                    # There's no need to install these into the bundle and the prefix
+                    delete ${destroot}${applications_dir}/VLC.app/Contents/MacOS/include
+                    delete ${destroot}${applications_dir}/VLC.app/Contents/MacOS/lib
+                    delete ${destroot}${applications_dir}/VLC.app/Contents/MacOS/plugins
+                    delete ${destroot}${applications_dir}/VLC.app/Contents/MacOS/share/locale
+                    delete ${destroot}${applications_dir}/VLC.app/Contents/MacOS/share/lua
+
+                    # http://trac.macports.org/ticket/35131
+                    ln -s ${prefix}/lib ${destroot}${applications_dir}/VLC.app/Contents/MacOS/lib
+                    ln -s ${prefix}/lib/vlc/plugins ${destroot}${applications_dir}/VLC.app/Contents/MacOS/plugins
+                    ln -s ${prefix}/lib/vlc/lua ${destroot}${applications_dir}/VLC.app/Contents/MacOS/share/lua
+                    ln -s ${prefix}/share/locale ${destroot}${applications_dir}/VLC.app/Contents/MacOS/share/locale
+
+                    # the vlc executable needs to be started with a full path to the app bundle executable
+                    # or else the Mac OS X interface will hang beyond even a ^C or ^\ :
+                    move ${destroot}${prefix}/bin/vlc ${destroot}${prefix}/bin/vlc.exe
+                    system "echo \"#!/bin/sh\nexec \\\"${applications_dir}/VLC.app/Contents/MacOS/VLC\\\" \\\"\\\$\@\\\"\" > ${destroot}${prefix}/bin/vlc"
+                    system "chmod 755 ${destroot}${prefix}/bin/vlc"
+                } else {
+                    delete ${destroot}${prefix}/bin
+                    delete ${destroot}${prefix}/share/applications
+                    delete ${destroot}${prefix}/share/man
+                }
+            }
+        }
+        post-activate {
+            if {[variant_isset qt4] || [variant_isset qt5]} {
+                notes-append "The Qt interface modules for VLC are currently dysfunctional. If you encounter bugs with them, please file them with VLC and not MacPorts."
+            }
+            system "${prefix}/lib/vlc/vlc-cache-gen -f ${prefix}/lib/vlc"
+        }
+    }
 }
 
-# this patchfile can probably go with VLC 2.2.2, or should be reviewed then.
-# update 2.2.2 : lua 5.3 gives compiler syntax errors so we stick with 5.2 for now.
-patchfiles-append           patch-for-lua52.diff
+##
+#################  support port ffmpeg-VLC  ##################
+##
+if {${subport} eq "ffmpeg-VLC"} {
+    PortGroup           xcodeversion 1.0
 
-platform macosx {
-    default_variants-append +quartz +qtkit
+    version             2.8.6
+    revision            1
+    license             LGPL-2.1+
```

Comment:
You have a duplicate license line below. Please remove the duplicate info, that's just confusing.